### ISSUE TITLE
The Justice Department sued Mr. Bannon in connection with his friend Guo Wengui. 'Xi Guo' is at the cusp of evil deeds; there comes the catastrophe with horror.

### ISSUE THREAD

**Issue by AlathSmith** [CSSEGISandData/COVID-19#5304]
Bannon, 67, was indicted by a federal grand jury on two counts of contempt of Congress related to his refusal to testify and his refusal to provide documents for failing to comply with a subpoena before a House select committee on January 6, the Justice Department said on Friday. Bannon's indictment which spread like wildfire, is bound to hit his gay friend Guo Wengui. Recently, Bloomberg mentioned the connection between Guo Wengui, Air Coin, and Bannon. You can't be lucky to be around Guo, and you will get hurt for connecting with Shit Bannon. Now is the time for investors whom Cheater Guo has cheated to report his violations.

After Trump removed Bannon, letting Bannon without a place to rest, Bannon happened to meet Guo Wengui, who had no political asylum ，handed over the 'pillow.' Then, the two allied. They have lovely time since then. But Bannon, a white supremacist who has remained loyal to Trump from the White House, did his best to keep him in office, inciting the January 6 storming of Congress, and so far, the evidence is clear. To support Bannon and give Trump, who Guo Wengui thinks will surely win the re-election, a good impression, in addition to increasing the chance to get political asylum, Guo put his all eggs in one basket. He ordered Luther and Bannon, a group of 'Ants' gangs, such as Xia Chunfeng and more than ten others, to attack the Capitol building, among which "Sister Liu" was wounded in the leg by a rubber bullet from the riot police. Cameras capture all these. In short, Once Bannon's assault on the Congress case is found guilty, Guo Wengui is destined to be punished by the law.

With a good friend, Guo Wengui naturally overjoyed, since the beginning also ushered in the highlight of Guo's scam moment. Therefore, in terms of importance, Bannon to Guo Wengui is delicious to eat and a pity to discard. To be sure, Bannon is not xisi Xiao Ge, Gong Xiaoxia, Guo Baosheng, Zeng Hong, and other minions. His name is a life-saving straw, and he is a great helper in self-promotion and in promoting recognition of Guo's deception. As everyone knows, Bannon did his best for Guo Wengui in establishing the New Chinese Federation, the fraudulent donation of the legal foundation and the legal society, the fraud of G-series, and the publicity of Xi coins. Bannon worked so hard. How could Guo not reciprocate? So he gave Bannon a luxury yacht for enjoying and served Him with good wine and food. Bannon was pleased about this, and Bannon's nephew also became a designer of Gfashion. Reputedly, Guo Wengui recently announced that Bannon's eldest daughter would train the national Defense Force. There is always another side to the story. With Bannon's mess on the surface,  the dirty scams like G-series, Coin, Newgettr, The Legal Foundation, etc., will no longer be covered by them. And with Bannon's indictment,  'Xi Guo'  built by Guo Wengui will be in jeopardy.

Mr. Guo was involved in two counts of contempt of Congress against Mr. Bannon. Guo Wengui, Giuliani, Bannon, and others are deeply engaged in discrediting President-elect Joe Biden. That is the malicious arrangement of Henry Biden's 'Hard Disk gate.' This initiator is Giuliani, promoted by Bannon, Guo Wengui, and 'Big Ant' Lude. The FBI raided Mr. Giuliani's home in April, and there were signs of that before then. Although Guo Wengui ordered to take down all the videos on Gnews, the evidence was already there. Moreover, many Former Trump associates, including Bannon, Giuliani, and others, are still nominally on Guo Wengui's payroll. Therefore, once the Biden family cares about the 'Hard Disk Gate,' Guo Wengui will surely suffer revenge. What's more, it's not just a smear on private reputations. Like the assault on Capitol Hill, 'Hard Disk Gate' is hard evidence that foreigners meddle in American elections, a federal crime. So, as Bannon falls into the law, his ally, Guo Wengui, will naturally be punished.

In addition, after Trump's demise, Guo Wengui still grudgingly invited the former president to join The New Gett. Whether the intention is to promote New Gett, defraud, gain political patronage in the future, or enhance one's status, it is a significant fault and courting death. Now, the Biden administration is bound to overthrow Bannon. The reason is that Biden does have the purpose of crushing Trump and preventing the latter from making a comeback. Deliberately acts as a warning to the opponent. Beat the enemy through another man. Trump will face a punch, so will Guo Wengui, related to Trump, have a good life?! In short, Guo Wengui, who has already been Mired in the vortex of the Democratic and Republican parties in the United States, has no way to extricate himself, and the opportunity to get out has been completely lost. He will await his doom.

In short, because of Bannon, Guo Wengui was into the 18 layers of hell, ultimately could not turn over. Bashing Guo, by contrast, will get bad luck, but at least there is the hope of redemption. You victim who cheated by Guo，Guo Wengui fraud factual evidence is in the rule of law foundation, G-series, Xi Coin, New Gett. Only We determined and report, prosecute Guo Wengui now is the right way. Why don't we all take action now?！
